### PR TITLE
Large numbers should be written with an underscore

### PR DESCRIPTION
**Pull request by zpeters**
may need a manual merge after previous PR

### CODE REVIEW

File: client_lib/client.exs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/elixir
-# command = "openssl s_client -connect localhost:41043 -showcerts -msg -servername local -tls1_2 -tlsextdebug -curves secp256k1 -cert device_certificate.pem"
+# command = "openssl s_client -connect localhost:41_043 -showcerts -msg -servername local -tls1_2 -tlsextdebug -curves secp256k1 -cert device_certificate.pem"
```
Comment (dominicletz) [inline client_lib/client.exs:2]: This change in the comment is wrong. The openssl command line would not understand the _ format.

File: config/config.exs [hunk 2]
```diff
@@ -14,7 +14,7 @@ config :logger,
 
 case Mix.env() do
   :test ->
-    System.put_env("KADEMLIA_PORT", "51053")
+    System.put_env("KADEMLIA_PORT", "51_053")
```
Comment (dominicletz) [inline config/config.exs:17]: This is in an environment variable and would also not be understood. Needs to stay without _

File: lib/diode.ex [hunk 3]
```diff
@@ -179,9 +179,9 @@ defmodule Diode do
   def seeds() do
     get_env(
       "SEED",
-      "diode://0xa3f06917a9a4846d44d39ae71ddbd69b4c0b1a1a@seed-alpha.diode.io:51053 " <>
-        "diode://0x0ffc572a936a1e0ebf9c43aacb145d08847f0a1d@seed-beta.diode.io:51053 " <>
-        "diode://0x0ffc572a936a1e0ebf9c43aacb145d08847f0aee@seed-gamma.diode.io:51053"
+      "diode://0xa3f06917a9a4846d44d39ae71ddbd69b4c0b1a1a@seed-alpha.diode.io:51_053 " <>
+        "diode://0x0ffc572a936a1e0ebf9c43aacb145d08847f0a1d@seed-beta.diode.io:51_053 " <>
+        "diode://0x0ffc572a936a1e0ebf9c43aacb145d08847f0aee@seed-gamma.diode.io:51_053"
```
Comment (dominicletz) [inline lib/diode.ex:184]: These are all strings, that will be parsed later. The port should not receive the underscore here.

File: lib/network/rpc.ex [hunk 4]
```diff
@@ -329,7 +329,7 @@ defmodule Network.Rpc do
         result(id, true)
 
       "net_version" ->
-        result(id, "41043")
+        result(id, "41_043")
```
Comment (dominicletz) [inline lib/network/rpc.ex:332]: Also this is a string not a number. Pls don't change.

File: lib/network/server.ex [hunk 5]
```diff
@@ -187,7 +187,7 @@ defmodule Network.Server do
       verify_fun: {&check/3, nil},
       fail_if_no_peer_cert: true,
       # Requires client to advertise the same
-      # openssl s_client -curves secp256k1 -connect localhost:41043 -showcerts -msg -servername local -tls1_2 -tlsextdebug
+      # openssl s_client -curves secp256k1 -connect localhost:41_043 -showcerts -msg -servername local -tls1_2 -tlsextdebug
```
Comment (dominicletz) [inline lib/network/server.ex:190]: Also string. Not change

**Review by dominicletz — CHANGES_REQUESTED** [review]
I've checked that changes and saw a couple that made changes to strings. Please review and update those areas. Strings should not be converted to use the underscore _, since they interact with external systems (openssl in on instance and json response in another instance).

**Comment by zpeters** [comment]
no problem, thank you!

**Pull request closed without merging**